parts/k8s/kubernetesconfigs.sh

```diff
@@ -242,10 +242,9 @@ function ensureDocker() {
     DOCKER_JSON_FILE=/etc/docker/daemon.json
     wait_for_file 1200 1 $DOCKER_JSON_FILE || exit $ERR_FILE_WATCH_TIMEOUT
     systemctlEnableAndStart docker
-    retrycmd_if_failure 6 1 10 docker pull busybox # pre-pull busybox, but don't exit if fail
```

Comment:
@jackfrancis since this line was removed does that mean that it should also be removed from #3781 ?

Reply:
It is removed from there :)

If you mean should we remove it from the VHD image, I actually think having a local busybox is super convenient (though of course it's a super small image by design).

Reply:
sounds good